components/helm-broker/internal/config/controller_config.go

```diff
@@ -0,0 +1,83 @@
+package config
```

Comment:
`ControllerConfig` and `Config` structs are almost the same (method `Load` is identical). Common fields:
```
Logger logger.Config
TmpDir string
Namespace string
Storage []storage.Config `valid:"required"`
ClusterServiceBrokerName string
DevelopMode bool
KubeconfigPath string `envconfig:"optional"`
```
extra fields in `Config` struct:
```
Port int `default:"8080"`
Helm helm.Config `valid:"required"`
HelmBrokerURL string
```
of which `HelmBrokerURL` is not used, `Port` has default value and `helm.Config` has 5 fields of which 4 has default value.
Extra fields in `ControllerConfig`:
```
ServiceName string
```
Can we make one file instead two almost identical?